netstate/auth/process_api_key.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package auth
+
+import (
+	"crypto/subtle"
+	"net/http"
+
+	"github.com/spf13/viper"
+)
+
+// InitializeHeaders : mocks HTTP headers to preset X-API-Key
+func InitializeHeaders() *http.Header {
```

Comment:
This appears to be only for testing purposes. If that's the case this should be moved into a test file.